spec.yml

```diff
@@ -0,0 +1,34 @@
+requirements:
+  base:
+    image: debian:stretch
+    pkg-manager: apt
+  fsl:
+    version: 5.0.10
+  afni:
+    version: latest
+  miniconda:
+    conda_install: [python=3.7, nipype]
+
+nfm_test:
+  data:
+    type: workflow_path
+    location: test/data
+  scripts:
+    test/scripts
+  env:
+    base:
+    - {image: debian:stretch, pkg-manager: apt}
+    fsl:
+    - {version: 5.0.10}
+    afni:
+    - {version: latest}
+    miniconda:
+    - {conda_install: [python=2.7, nipype]}
```

Comment:
I think in the long run it would be nice to make environments that are only differences from `requirements`, but no objections to being explicit like this, either.

Reply:
this is a good idea! not for testkraken itself, but for niflow that will be useful